scripts/groundedattackscript/groundedattackscript.gml

```diff
@@ -1,7 +1,7 @@
 /// @function                   GroundedAttackScript(moveToDo, onGround);
 /// @param {moveToDo}  message  The message to show
 
-function GroundedAttackScript(moveToDo, onGround, gravityMult, fallingMult) 
+function GroundedAttackScript(moveToDo, onGround, gravityMult, fallingMult, _ignoreWalkoff) 

```

Comment:
Why does this parameter randomly start with an underscore? Remove the underscore here.